Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1528,6 +1528,17 @@ void Renderer::SwapImpl(AbstractTexture* texture, const EFBRectangle& xfb_region
   ClearEFBCache();
 }
 
+void Renderer::OnSwapInvalid()
+{
+  // ensure all commands are sent to the GPU.
+  // Otherwise the driver could batch several frames togehter.
+  glFlush();
+   // Clean out old stuff from caches. It's not worth it to clean out the shader caches.
+  g_texture_cache->Cleanup(frameCount);
+   // Invalidate EFB cache
+  ClearEFBCache();
```

Comment:
Why does the EFB cache need to be invalidated? The framebuffer doesn't change when the "scanout" occurs.